packages/netlify/test/functions/fixtures/includes/astro.config.mjs

```diff
@@ -0,0 +1,11 @@
+import netlify from '@astrojs/netlify';
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  output: 'server',
+  adapter: netlify({
+    includedFiles: ['files/**/*.csv', 'files/include-this.txt'],
+    excludedFiles: ['files/subdirectory/not-this.csv'],
```

Comment:
Can we create tests cases where only `includedFiles` and `excludedFiles` are provided? E.g.

```js
adapter: netlify({
	includedFiles: [""]
})


adapter: netlify({
	excludedFiles: [""]
})
```

Reply:
Added a new test fixture to show excluding traced node modules